jenkins-pipelines/manager/centos-manager-sanity.jenkinsfile

```diff
@@ -16,5 +16,5 @@ managerPipeline(
     post_behavior_loader_nodes: 'destroy',
     post_behavior_monitor_nodes: 'destroy',
 
-    downstream_jobs_to_run: 'centos-upgrade-test, centos-sanity-ipv6-test, sct-feature-test-backup, sct-feature-test-backup-azure, sct-feature-test-backup-gce'
+    downstream_jobs_to_run: 'centos-upgrade-test, centos-sanity-ipv6-test, sct-feature-test-backup, sct-feature-test-backup-azure, sct-feature-test-backup-gce, centos-upgrade-meta-installation-test'
```

Comment:
will the other, regular upgrade tests still work?

Reply:
Of course

Reply:
seems like something is not clear... this commit lacks of information about the feature (like why it was introduced) and why a new upgrade method is being added, if it will not be supported once this version will not be supported anymore

Reply:
Don't invest time on it because I'm not going to merge it.